Documentation/proxy.md

```diff
@@ -145,6 +145,9 @@ If you are running etcd under systemd, you should modify the service file with c
 sudo systemd restart etcd
 ```
 
-If you see an error, you can read the [add member troubleshooting doc](runtime-configuration.md#error-cases).
+If you see an error, you can read the [add member troubleshooting doc][runtime-configuration].
```

Comment:
If an error occurs, check the [add member troubleshooting doc][runtime-configuration].